2018-09-10-was-ist-das-xcamp-und-wer-sollte-teilnehmen.md

```diff
@@ -0,0 +1,18 @@
+---
+title:     Video-Interview: Was ist das XCamp und wer sollte teilnehmen?
+author:    Jens Bothmer
+published: 10.09.2019
+image:     
+tags:      XCamp2019, Sessions
+---
+
+Interview mit Dr. Christiane Gerigk und Jens Bothmer – Initiatoren des Xcamps.
+
+Termin 13.09.2019
+Frankfurt am Main
+Skydeck Silberturm, 31. Etage
+bei DB Systel GmbH
```

Comment:
These lines will not wrap but will be joined to a long line. To get line breaks, you need either three blanks at the end of each line (ugly, because invisible), or you need to wrap the lines in a `<pre>` tag (formatted as fixed width chars then, also not too nice), or add `<br>` tags at the end of each line explicitly (my favorite).

Reply:
Also see https://github.com/showdownjs/showdown/wiki/Showdown's-Markdown-syntax#paragraphs for background info